phobos/simple_deployment.md

```diff
@@ -0,0 +1,182 @@
+# Install rpc-ceph in the Phobos lab OnMetal 
+
+
+## Create Environment
+* 1 admin vm
+* 1 rsyslog vm
+* 5 boxes for storage nodes
+
+```bash
+openstack server create --flavor  vm-osa-aio --image a4a9bc69-de27-4cec-903a-cf57be693738 \
+  --nic net-id=8dd6207d-a1f5-4dfe-b97d-a8c967eb8548  --security-group 58753c65-1804-4eee-8349-77f1c5573001 \
+  --key-name letterjkey ceph-admin01-c_
+
+openstack server create --flavor  vm-osa-aio --image a4a9bc69-de27-4cec-903a-cf57be693738 \
+  --nic net-id=8dd6207d-a1f5-4dfe-b97d-a8c967eb8548  --security-group 58753c65-1804-4eee-8349-77f1c5573001 \
+  --key-name letterjkey rsyslog01-c_
+
+openstack server create --flavor ironic-rpc-storage-interim  --image 78856fa6-1de4-42d3-b545-466f6501e21f \
+  --nic net-id=8dd6207d-a1f5-4dfe-b97d-a8c967eb8548  --security-group 58753c65-1804-4eee-8349-77f1c5573001 \
+  --key-name letterjkey ceph-stor01-c_
+
+openstack server create --flavor ironic-rpc-storage-interim  --image 78856fa6-1de4-42d3-b545-466f6501e21f \
+  --nic net-id=8dd6207d-a1f5-4dfe-b97d-a8c967eb8548  --security-group 58753c65-1804-4eee-8349-77f1c5573001 \
+  --key-name letterjkey ceph-stor02-c_
+
+openstack server create --flavor ironic-rpc-storage-interim  --image 78856fa6-1de4-42d3-b545-466f6501e21f \
+  --nic net-id=8dd6207d-a1f5-4dfe-b97d-a8c967eb8548  --security-group 58753c65-1804-4eee-8349-77f1c5573001 \
+  --key-name letterjkey ceph-stor03-c_
+
+openstack server create --flavor ironic-rpc-storage-interim  --image 78856fa6-1de4-42d3-b545-466f6501e21f \
+  --nic net-id=8dd6207d-a1f5-4dfe-b97d-a8c967eb8548  --security-group 58753c65-1804-4eee-8349-77f1c5573001 \
+  --key-name letterjkey ceph-stor04-c_
+
+openstack server create --flavor ironic-rpc-storage-interim  --image 78856fa6-1de4-42d3-b545-466f6501e21f \
+  --nic net-id=8dd6207d-a1f5-4dfe-b97d-a8c967eb8548  --security-group 58753c65-1804-4eee-8349-77f1c5573001 \
+  --key-name letterjkey ceph-stor05-c_
+```
+
+
+## Storage Nodes favor ironic-rpc-storage-interim additional drives
+/dev/sdb | 3TB
+...
+/dev/sdk | 3TB
+
+
+## Phobos Xenial image is set to not allow root user to log in
+To remove the restriction run this on each osd node and on rsyslog
+```bash
+ssh ubuntu@<ip>  "sudo sed -i -e 's/^no-port-forwardin.*\(ssh-rsa.*$\)/\1/g' /root/.ssh/authorized_keys"
```

Comment:
Isn't this on a key-by-key basis? I guess you are removing it from your initial key to allow a login, so that seems like it's ok.

Reply:
The current Xenial image used in phobos has a string placed in front of the key in the /root/.ssh/authorized_keys file and this sed statement removes that so you can log into the host as root.

Reply:
:+1: makes sense